config/src/main/java/com/typesafe/config/impl/ConfigImplUtil.java

```diff
@@ -32,6 +32,14 @@ else if (a == b) // catches null == null plus optimizes identity case
             return a.equals(b);
     }
 
+    public static boolean isC0Control(final char ch) {
```

Comment:
I had originally thought it was needed in ConfigUtil, but realized that was wrong - I'll remove public. Looks like we only ever use the codepoint version, I'll remove the char version.